components/leaderboard/profiles.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+interface Person {
+  id: number;
+  initials: string;
+  name: string;
+  role: string;
+  username: string;
+  totalLeads: number;
+}
+
+const Profiles = ({ Leaderboard }: any) => {
+  return <div>{Item(Leaderboard)}</div>;
```

Comment:
This is not any bug but we should not call a component like function. Here we can replace it as follows:
```javascript
return <div><Item data={Leaderboard}/></div>;
```

Reply:
fixed